src/akshita/Vowels.java

```diff
@@ -0,0 +1,36 @@
+//WAP to get the sum of digits n a string only if there is any vowels in string else return 0.
+//Input: "Pu1ne5lnd3ia" , "R1hlTns7h";
+//Output:9 
+
+package akshita;
+
+public class Vowels {
+	int sum(String input) {
+		boolean test = false;
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') {
+				test = true;
```

Comment:
here you should break, as per the requirement **if there is any one vowel**
hope you got the point